packml_smach/src/executeState.py

```diff
@@ -33,7 +37,13 @@ def __init__(self):
         
     def execute(self, userdata):
         rospy.loginfo('Executing state Execute')
-        if userdata.execute_in == True:
+
+        # Receive Stop Tag
+        var4 = config.client.get_node("ns=3;s=\"PackML_Status\".\"UN\".\"Cmd_Stop\"")  
+        var4.set_value(True)
+        config.stopTag = var4.get_value()
+        print("Received stop command: ", config.stopTag)
```

Comment:
I will receive "True" - the value I set. This is just to give a name to the value of var variable, ease the usage in the next two lines. I notice it doesn't need to be config.stopTag, just stopTag will be enough.